cookbook/email/gmail.rst

```diff
@@ -75,3 +75,13 @@ You're done!
 
     The ``gmail`` transport is simply a shortcut that uses the ``smtp`` transport
     and sets ``encryption``, ``auth_mode`` and ``host`` to work with Gmail.
+
+.. note::
+
+    Depending on your Gmail account settings, you may get authentication errors within your app.
+    If your Gmail account uses 2-Step-Verification, you should `generate an App password`_ to use for your
+    ``mailer_password`` parameter.
```

Comment:
There is no need for this line break. Can you also please wrap lines after the first word that crosses the 72nd character instead?